### DISCUSSION TITLE
A group of bees is called a what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#138] [Galaxy Trivia] [answered]
**Hint:** 🚳

**Comment by Tynab** [comment]


**Reply by ParanoidUser** [reply]
Swarm is definitely the most generic way to refer to a group of bees, but there is another old English word (see hint). Can you guess?

**Comment by sugandha27** [comment]
colony ?

**Reply by ParanoidUser** [reply]
Nice try! The clue refers to a street sign with a bicycle. The correct answer would be a shortened form of it. Surprisingly, in Old English it means a swarm/colony/hive of bees. Can you guess?

**Comment by SANKARAMDAS** [comment]
A group of bees is called a hive, a swarm, or a colony.

**Reply by ParanoidUser** [reply]
Nice try! But it doesn't match the hint that was given. Any other ideas? 🤔

**Comment by peterbaccaro** [comment, marked as answer]
There are many terms for a group of bees but I think the term you are looking for is **Bike of Bees**.

The reason this is a collective noun for bees is because the word “bike” is an old English word which means a colony, nest, or swarm.
